budgie-fuzzyclock/fuzzyclock/FuzzyClockApplet.vala

```diff
@@ -0,0 +1,481 @@
+/*
+ * This file is part of budgie-desktop
+ *
+ * Copyright © 2014-2019 Budgie Desktop Developers
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ */
+
+public class FuzzyClockPlugin : Budgie.Plugin, Peas.ExtensionBase
+{
+    public Budgie.Applet get_panel_widget(string uuid)
+    {
+        return new FuzzyClockApplet();
+    }
+
+}
+
+enum ClockFormat {
+    TWENTYFOUR = 0,
+    TWELVE = 1;
+}
+
+public const string CALENDAR_MIME = "text/calendar";
+
+public class FuzzyClockApplet : Budgie.Applet
```

Comment:
General comment - please add a small comment above each method to say what is is, the params it uses, return value (what it signifies and possible values)